src/game/SpellAuras.cpp

```diff
@@ -5724,6 +5724,19 @@ void Aura::PeriodicTick()
                         }
                         break;
                     }
+                    case 46771: {                               // Flame Sear
+                        if (target->GetTypeId() == TYPEID_PLAYER) {
+                            if (!target->HasAura(45343) &&
+                                !target->HasAura(47300)) {          // No Dark Flame applied. There are two auras
+                                if (m_periodicTick % 4 == 1) {      // Only doze the aura every 2 ticks (should maybe be one second cooldown)
+                                    target->CastSpell(target, 45348,
```

Comment:
The check currently does "if no dark flame aura and if no fire flame aura do stuff", afaik u want "if dark flame aura procs do stuff, if currently has flame touched, run immunity" "if fire flame aura procs, do stuff, if currently has shadow touched, run immunity" or am I wrong?